src/System.Private.Xml/src/System/Xml/Serialization/XmlSerializer.cs

```diff
@@ -127,13 +127,13 @@ internal enum SerializationMode
 
         internal static SerializationMode Mode { get; set; } = SerializationMode.ReflectionAsBackup;
 
-        private static bool ReflectionMethodEnabled
-        {
-            get
-            {
-                return Mode == SerializationMode.ReflectionOnly || Mode == SerializationMode.ReflectionAsBackup;
-            }
-        }
+//        private static bool ReflectionMethodEnabled
```

Comment:
The name of this suggests it might be used by code somewhere via reflection.  Do we know it's not?